<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit</class>
 <widget class="QDialog" name="deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_accured_interest">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>230</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Срок вклада:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>320</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Сумма налога:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_deposit_term">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Какой месяц:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>430</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 24pt &quot;Cochin&quot;;</string>
   </property>
   <property name="text">
    <string>Капитализация процентов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_partial_replanishment_month">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>месяц</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_partial_withdraw_month">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>560</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tax_ammount">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>320</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Сумма депозита</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Сумма:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_deposit_end">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>120</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Сумма:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_periodicity_of_payments">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Периодичность выплат,</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>230</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Начисленные проценты:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tax_rate">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Налоговая ставка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Процентная ставка:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_interes_rate">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Сумма депозита:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Список частичных изъятий</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>каждый</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_replenishments_sum">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Какой месяц:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_withdraw_sum">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>510</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_deposit_amount">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 30pt &quot;Cochin&quot;;
background-color: rgb(81, 96, 76);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>в конце срока:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
border-radius: 20px;
font: 20pt &quot;Cochin&quot;;
background-color: rgb(158, 140, 136);</string>
   </property>
   <property name="text">
    <string>Список пополнений</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_credit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>500</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid white;
   border-radius: 20px;
   font: 20pt &quot;Cochin&quot;;
   background-color: rgb(81, 96, 76);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #112617, stop: 1 #112617);
    color: white;
    border: 2px solid rgb(81, 96, 76);
}</string>
   </property>
   <property name="text">
    <string>Count</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
